packages/katello/katello/katello-remove

```diff
@@ -3,15 +3,25 @@
 require 'optparse'
 require 'fileutils'
 
+options = {}
 optparse = OptionParser.new do |opts|
-  opts.banner = "This script is used to remove all katello related packages and services. \nThis should only be used if you are sure you are wanting to completly remove Katello from this machine and lose all of your settings and synced content."
+  opts.banner = "This script is used to remove all Katello related packages and services. \nThis should only be used if you are sure you are wanting to completly remove Katello from this machine and lose all of your settings and synced content."
 
+  options[:unattended] = false
+  opts.on( '--unattended', 'Remove without confirmation' ) do
+    options[:unattended] = true
```

Comment:
To follow convention with other tools (like yum) and katello scripts, I would call this `--assume-yes` and have a `-y` short option too

[Here is what we do in katello-restore]( https://git.io/vx5Rj)